src/main/java/com/napier/sem/DAO.java

```diff
@@ -53,20 +52,65 @@ public DAO(Connection connection)
     /**
      * Constructs the SQL query required and returns the result of the
      * query.
+     *
+     * @return An ordered list of countries in the world sorted by descending population
+     */
+    public ArrayList<Country> allCountries() {
+        // Define the SQL query as a string
+        String statementString = "SELECT code, name, continent, region, population, (\n" +
+                "    SELECT name\n" +
+                "    FROM city ci\n" +
+                "    WHERE countrycode = co.code\n" +
+                "        AND ci.id = co.capital\n" +
+                "    ) AS capital\n" +
+                "FROM country co\n" +
+                "ORDER BY population DESC\n";
+//              "LIMIT 20";
+
+        return ExecuteCountryStatement(statementString);
+    }
+
+    /**
+     * Constructs the SQL query required and returns the result of the
+     * query.
+     *
+     * @return An ordered list of countries in the world sorted by descending population
```

Comment:
maybe this should be "An ordered list of countries in a specified area sorted by descending population"?